widgy/contrib/widgy_mezzanine/views.py

```diff
@@ -49,7 +51,15 @@ def get_page_from_node(node):
 class PageViewMixin(object):
     def get_page(self):
         try:
-            return Page.objects.published(for_user=self.request.user).get(pk=self.kwargs['page_pk'])
+            qs = Page.objects.published(for_user=self.request.user)
+            if 'slug' in self.kwargs:
+                warnings.warn(
+                    "Referencing a page by slug in urls is deprecated. Use page_pk instead",
```

Comment:
I don't think this warning is useful the way it is written. I think it should say something:

```python
"""
Using slug when reversing widgy.contrib.widgy_mezzanine.views.handle_form or widgy.contrib.widgy_mezzanine.preview is deprecated. You should update your code to look like this:

    url = urlresolvers.reverse('widgy.contrib.widgy_mezzanine.views.preview', kwargs={
        'node_pk': node.pk,
        'page_pk': page.pk,
    })
"""
```